skops/card/tests/test_card.py

```diff
@@ -1887,3 +1887,22 @@ def test_toc_with_invisible_section(self, card):
         ]
 
         assert toc == "\n".join(exptected_toc)
+
+
+class TestCardSaveWithPlots:
+    def test_copy_plots(self, destination_path, model_card):
+        import matplotlib.pyplot as plt
+
+        plt.plot([4, 5, 6, 7])
+        plt.savefig(Path(destination_path) / "fig1.png")
+        model_card = model_card.add_plot(fig1="fig1.png")
+
+        plt.plot([7, 6, 5, 4])
+        plt.savefig(Path(destination_path) / "fig2.png")
+        model_card = model_card.add_plot(fig2="fig2.png")
+
+        model_card.save(Path(destination_path) / "README.md")
```

Comment:
So here, you're not using the new feature here, right? And the test still passes. This is because you saved the figures in the correct destination right from the start. But that's what we want the new feature to do.

Reply:
Yeah 🤦🏽 Sorry I missed that